org.sofproject.alsa.topo/src/org/sofproject/alsa/topo/model/AlsaTopoGraph.java

```diff
@@ -490,37 +490,16 @@ public void serialize() throws CoreException, IOException {
 		ByteArrayOutputStream os = new ByteArrayOutputStream();
 
 		Writer writer = new BufferedWriter(new OutputStreamWriter(os));
-		// tlv-s
 		tlvs.serialize(writer);
-
-		// vendor tokens
 		vTokensIndex.serialize(writer);
-
-		// vendor tuples
 		vTuplesIndex.serialize(writer);
-
-		// data
 		dataIndex.serialize(writer);
-
-		// control bytes
 		controlBytes.serialize(writer);
-
-		// control mixers
 		controlMixers.serialize(writer);
-
-		// pcm capabilities
 		pcmCapsIndex.serialize(writer);
-
-		// pcm-s
 		pcms.serialize(writer);
-
-		// be-s
 		beIndex.serialize(writer);
-
-		// hw-configs
 		hwConfigs.serialize(writer);
-
-		// pipelines (widgets + graphs)
```

Comment:
Looks like it was previously marked as your addition cos of spaces vs tabs issue. You can remove it now from this pr as it's not related.